README.pl.md

```diff
@@ -0,0 +1,144 @@
+<p align="center"><a href="https://newpipe.net"><img src="assets/new_pipe_icon_5.png" width="150"></a></p> 
+<h2 align="center"><b>NewPipe</b></h2>
+<h4 align="center">Wolny, lekki streamingowy frontend na Androida.</h4>
+
+<p align="center"><a href="https://f-droid.org/packages/org.schabi.newpipe/"><img src="https://fdroid.gitlab.io/artwork/badge/get-it-on-pl.svg" alt="Pobierz z F-Droid" height=80/></a></p>
+
+<p align="center">
+<a href="https://github.com/TeamNewPipe/NewPipe/releases" alt="GitHub release"><img src="https://img.shields.io/github/release/TeamNewPipe/NewPipe.svg" ></a>
+<a href="https://www.gnu.org/licenses/gpl-3.0" alt="License: GPLv3"><img src="https://img.shields.io/badge/License-GPL%20v3-blue.svg"></a>
+<a href="https://github.com/TeamNewPipe/NewPipe/actions" alt="Build Status"><img src="https://github.com/TeamNewPipe/NewPipe/workflows/CI/badge.svg?branch=dev&event=push"></a>
+<a href="https://hosted.weblate.org/engage/newpipe/" alt="Translation Status"><img src="https://hosted.weblate.org/widgets/newpipe/-/svg-badge.svg"></a>
+<a href="https://web.libera.chat/#newpipe" alt="IRC channel: #newpipe"><img src="https://img.shields.io/badge/IRC%20chat-%23newpipe-brightgreen.svg"></a>
+<a href="https://www.bountysource.com/teams/newpipe" alt="Bountysource bounties"><img src="https://img.shields.io/bountysource/team/newpipe/activity.svg?colorB=cd201f"></a>
+</p>
+<hr>
+<p align="center"><a href="#screenshots">Screenshoty</a> &bull; <a href="#description">Opis</a> &bull; <a href="#features">Funkcje</a> &bull; <a href="#installation-and-updates">Instalacja i aktualizacje</a> &bull; <a href="#contribution">Wkład</a> &bull; <a href="#donate">Wsparcie</a> &bull; <a href="#license">Licencja</a></p>

```

Comment:
None of these links work, as you can see [here](https://github.com/TeamNewPipe/NewPipe/blob/730bddb62ab60bc2dc3339e287f736dea9c1ada4/README.pl.md). GitHub automatically generates `id`s for headers depending on their title (so `## Screenshoty` below can be linked to with `#screenshoty` here), but since you translated that to Polish, these links won't work anymore.

There are two solutions:
1. Just update the links to use the Polish titles (in the link to the preview of the Polish README that I sent, if you hover on a title you should see a chain icon beside it, you can click on that icon and copy the URL in the URL bar).
2. Add spans with the English `id`s before each header, like this:
    ```
    <span id="screenshots"></span>
    ## Screenshoty
    ```

I prefer solution 1, but I don't mind 2 much.

Feel free to ask me if anything is unclear.

Reply:
You want to use hyphens (`-`) between words, not spaces. You could've saved yourself effort by using the method I mentioned though:

> in the link to the preview of the Polish README that I sent, if you hover on a title you should see a chain icon beside it, you can click on that icon and copy the URL in the URL bar

Reply:
I fixed the link.